CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Ewon Ignition 8 Connector Changelog
 
+## v1.1.12
+
+### Major Changes
+
+* Ewon tag descriptions are now synced to Ignition

```

Comment:
since this is customer facing should you include where they are stored in ignition and how you access them? (tooltip).

Reply:
or update the readme to document this functionality.

Reply:
I just edited the readme to say that Ewon tag description is synced to Ignition tag documentation and tooltip.